modules/gdscript/gdscript.cpp

```diff
@@ -2176,6 +2205,22 @@ GDScriptLanguage::~GDScriptLanguage() {
 	singleton = NULL;
 }
 
+void GDScriptLanguage::add_orphan_subclass(const String &p_compund_name, const ObjectID &p_subclass) {
```

Comment:
`compund` should be `compound`. But since we are introducing FQN, maybe we want to call it `p_qualified_name` to keep consistency (the same applies to other instances of this argument name).